.gitignore

```
@@ -1,6 +1,9 @@
 # Maven target directory
 /target/
 
+# Eclipse project information
+.project
```

Assuming `.project` is created when you clone the repository and import it as a project, I guess the file is created only in the same directory as this `.gitignore`. If so, I'd suggest limiting its scope, that is
```suggestion
/.project
```